### PR TITLE
passed division to the math.div function

### PR DESCRIPTION
**Pull request by MosDeef**

### CODE REVIEW

**Comment by bconfiden2** [comment]
Hi, @MosDeef !
Thanks for your PR, and I wonder how you came to know this repo(especially Issue #9),
since it is just a simple webpage for datamining lab of Kookmin Univ., Korea.

File: assets/vendor/bootstrap/scss/_variables.scss [hunk 1]
```diff
@@ -715,7 +715,7 @@ $nav-divider-margin-y:              $spacer / 2 !default;
 
 // Navbar
 
-$navbar-padding-y:                  $spacer / 2 !default;
+$navbar-padding-y:                  math.div($spacer, 2);
```
Comment (bconfiden2) [inline assets/vendor/bootstrap/scss/_variables.scss:718]: Replacing only with `math.div()` might result in the following error.

image

It is recommended to build and test locally.

**Comment by MosDeef** [comment]
> Hi, @MosDeef ! Thanks for your PR, and I wonder how you came to know this repo(especially Issue #9), since it is just a simple webpage for datamining lab of Kookmin Univ., Korea.

Hiya - I just came across it browsing the good-first-issue tags and looked like something I could handle

**Comment by bconfiden2** [comment]
Highly appreciate for your contribute @MosDeef !
But if you don't mind, can I assign this task for our lab members, to practice on git/github?

**Comment by MosDeef** [comment]
> Highly appreciate for your contribute @MosDeef ! But if you don't mind, can I assign this task for our lab members, to practice on git/github?

Not at all 😊

**Comment by bconfiden2** [comment]
Thanks ! 😄

**Pull request closed without merging**